Source/ChocolateyGui/ViewModels/Controls/LocalSourceControlViewModel.cs

```diff
@@ -183,6 +197,37 @@ public bool ReceiveWeakEvent(Type managerType, object sender, EventArgs e)
             }
         }
 
+        public async void ExportAll()
+        {
+            try
+            {
+                var fileStream = this._persistenceService.SaveFile("*.config", "Config Files (.config)|*.config");
+
+                if (fileStream == null)
+                {
+                    return;
+                }
+
+                using (StreamWriter sw = new StreamWriter(fileStream, Encoding.ASCII))

```

Comment:
Yip, this is just a PoC, to see what people's thoughts are.  Can switch to using Xml based classes for actual implementation.